src/components/NightMode/NightMode.tsx

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { FiMoon, FiSun } from 'react-icons/fi';
+import { useLocalStorage } from '../../util/useLocalStorage';
+
+export function NightMode() {
+  const containerFlexClass = 'flex justify-center items-center';
```

Comment:
No need it here if it does not change. Place it there!

Reply:
done